### ISSUE TITLE
FORD-generated doc changes even if FORTRAN code doesn't change

### ISSUE THREAD

**Issue by patflynngithub** [Fortran-FOSS-Programmers/ford#482]
When I run FORD a second time on unchanged FORTRAN code, the generated documentation changes. I don't mind regenerating all the doc files again, but, when I do a *git status* and a *git diff*, I get a lot of underlying changes in the doc even though it displays the same in the browser. This is also true when I make a minor change to the FORTRAN code.

To no avail, I've tried setting *parallel* to 0 and turning off the graphs.

**Comment by ZedThree** [comment]
I would normally recommend that generated files, like Ford docs, aren't kept under version control. If you're deploying the pages via something like Github Pages, I would probably suggest keeping the generated website in a separate branch and only deploying that.

I'm a bit surprised that things are changing order, I would've thought everything was fairly deterministic. I'll looking into sorting the collections.

**Comment by patflynngithub** [comment]
Yeah, as a response to this phenomenon, we've created a separate repo for hosting the project's HTML documentation (including the FORD-generated doc). We keep the Markdown-formatted doc on the project repo's wiki, which contains a link to the doc repo's GitHub Pages.
